mne/io/meas_info.py

```diff
@@ -226,14 +226,6 @@ def __repr__(self):
         st %= non_empty
         return st
 
-    def _anonymize(self):
```

Comment:
To keep Info as close to a dict as possible, I'm removing this private method and call the public function `anonymize_info` instead.